README.rst

```diff
@@ -2,80 +2,128 @@
 
 Trustlines Smart Contract Platform
 ==================================
+Introduction
+------------
 
-This documentation explains how to deploy Trustlines smart contracts,
-for creating your own currency network and trustlines exchange.
-The exchange functionality is not fully supported at this point.
-The current documentation is written for an Ubuntu Linux system.
+This repository contains the smart contracts implementing the Trustlines logic.
+This includes:
 
-Prerequisites
--------------
+- Currency Networks
+- Exchanges
+- Identity implementation, proxy, and proxy factory
+
+It also includes deploy tools that can be used to deploy these contracts.
```

Comment:
I would write `deploy command tools` or cli tools to make clear what kind of tools that are. 
Or we could write deploy tools with cli and python interface, to show that you can access them from the cli and from python.